css/style.css

```diff
@@ -0,0 +1,651 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  scroll-behavior: smooth;
+  min-height: 100%;
+}
+
+body {
+  min-width: 375px;
+  background-color: #fff;
+  font-family: 'Lato', sans-serif;
+  color: #fff;
+}
+
+/* ------------------------------- Header Section ----------------------------- */
+
+header {
+  margin: 0;
+  top: 0;
+}
+
+#homepage {
+  background-image: url(/images/hero-image-mobile.png);
+  background-repeat: no-repeat;
+  height: 90vh;
+  background-position: center;
+  background-size: cover;
+}
```

Comment:
- [ ] [OPTIONAL] The background image you tried to apply, does not show on your application especially when I use other services to open HTML files other than Vscode. Kindly consider accessing images using `./images/` since we are accessing images at other levels of the file path. Consider reading this [documentation](https://www.w3schools.com/html/html_filepaths.asp) for more information.